### ISSUE TITLE
[IDEA] Make keyboard shortcuts easier to discover

### ISSUE THREAD

**Issue by saqimtiaz** [TiddlyWiki/TiddlyWiki5#4958]
In #4909 Jeremy suggested that: "There's a nice feature on the iPad where you can hold down a modifier key to see a HUD of all the key combinations that can be used. I wonder if we could do something similar: a global message handler that collects the active keyboard shortcuts and displays the HUD. We wouldn't be able to trigger it with a modifier key, we'd have to use F1 or ctrl-?. (It would be tricky to use a button to trigger it because the act of clicking the button will move the focus away)."

As we have added a fair few keyboard shortcuts to TW recently, a feature that makes it easier to discover and be reminded of keyboard shortcuts warrants consideration.

**Comment by rmunn** [comment]
In addition to context-sensitive keyboard shortcuts, it might be nice if that HUD also had a link or button to open the Control Panel tiddler with the Keyboard Shortcuts tab already selected, so that the user can more easily discover all the TiddlyWiki keyboard shortcuts as well as the ones applying to the current context. (Is it possible to create a link to the Control Panel with a specific tab selected, or would it have to be a link to the Control Panel tiddler, and then the user would have to click on Keyboard Shortcuts by themselves?)

**Comment by saqimtiaz** [comment]
@rmunn good idea and that should be doable. Selected tabs are controlled by state tiddlers so we can have a button that sets the appropriate state tiddlers and opens the Control Panel.

**Comment by rmunn** [comment]
https://github.com/Jermolene/TiddlyWiki5/issues/4957#issuecomment-720980153 is also relevant to the discoverability of keyboard shortcuts.

**Comment by MrDowntempo** [comment]
#5017 is another situation where a keyboard shortcut's discoverability (ESC) could help alleviate the issue.
